cmd/check.go

```diff
@@ -17,125 +17,105 @@ import (
 	"github.com/open-policy-agent/opa/util"
 )
 
-var checkParams = struct {
+type checkParams struct {
```

Comment:
There hadn't been any tests on cmd/check.go go before -- so I had to restructure things bit there, taking `cmd/build.go` as a model.

Nothing of significance has changed in this file, except for putting `WithCapabilities()` onto the parser code paths...